crates/common/log_manager/src/actor.rs

```diff
@@ -0,0 +1,154 @@
+use easy_reader::EasyReader;
```

Comment:
The file name `actor.rs` is very misleading. The functions here do nothing with actor. I would suggest `log_utils.rs` or something that explains the purpose of the file.